src/Controller/ImageStyleDownloadController.php

```diff
@@ -6,7 +6,6 @@
 use Drupal\Core\Image\ImageFactory;
 use Drupal\Core\Lock\LockBackendInterface;
 use Drupal\image\ImageStyleInterface;
-use Drupal\system\FileDownloadController;
```

Comment:
This removal causes the image style download controller to inherit from our new (fixed) file download controller rather than core's. Note that I'm not using image styles and haven't tested them--before or after this change.